pypureclient/flasharray/FA_2_0/client.py

```diff
@@ -21,7 +21,7 @@ class Client(object):
     DEFAULT_TIMEOUT = 15.0
     DEFAULT_RETRIES = 5
     # Format: client/client_version/endpoint/endpoint_version/system/release
-    USER_AGENT = USER_AGENT_TEMPLATE.format(prod='FA', sys=platform.system(), rel=platform.release())
+    USER_AGENT = USER_AGENT_TEMPLATE.format(prod='FA', rest_version='2.0', sys=platform.system(), rel=platform.release())
```

Comment:
> LGTM but I cannot tell how this fixes the documentation issue. Can you elaborate in the commit message?]

This change fixes the integration. Previous commit introduced `{rest_version}` part in USER_AGENT_TEMPLATE and `rest_version` format option in `client.py` but _only_ in Pure1 part.

It does not affect py-pure-client pypi package, but read-the-doc has it is own way of docs generation. It imports files directly from repo, so having unresolved part in template lead to the import error which in turn broke documentation build.